src/app.ts

```diff
@@ -280,6 +281,7 @@ export const start = async (args) => {
   });
 
   branchTable.key("space", function () {
+    // @ts-ignore
     const selection = parseSelection(this.items[this.selected].content);
```

Comment:
This looks like a fun one to solve in TS 😂 I couldn't find how/where `this` is handled

Reply:
I _think_ `this` is supposed to refer to the `branchTable` variable which is of type `Widgets.ListTableElement` from `blessed`. However, this type doesn't actually publish the `items` or `selected` properties, so we'll probably just have to make our own type since we _know_ it's there.

I'll give it a shot and push an update.

Reply:
Thanks - I'm okay with TS ignore for it or deferring till later if @jwu910 is.

Reply:
Just pushed an update, take a look and lmk what you think.